bin/run_tests.py

```diff
@@ -35,8 +37,8 @@ def run(test_outputs, testDir, tests=None):
     makedirs(dirDict["out"])
     dirDict["plots"] = osp.join(dirDict["out"], "plots")
     makedirs(dirDict["plots"])
-    run_test_sims(runInfo, dirDict, pflag)
-    if not tests:
+    run_test_sims(tests, dirDict, pflag)
+    if runInfoAnalyt:
```

Comment:
This will not work if runInfoAnalyt does not exists (it will throw an error). So running seperate tests will not work

This will work:
```
try:
  run_test_sims_analyt(runInfoAnalyt, dirDict)
except:
  pass
```

Reply:
Good catch...pun intended